test/testgarbage.cpp

```diff
@@ -1658,7 +1658,7 @@ class TestGarbage : public TestFixture {
     }
 
     void garbageCode213() { // #8758
-        ASSERT_THROW(checkCode("{\"\"[(1||)];}"), InternalError);
```

Comment:
spontanously it seems unfortunate that Cppcheck will not throw an exception anymore for this code.. is there a good reason to let this through?